openshift/release/download_release_artifacts.sh

```diff
@@ -1,45 +1,54 @@
 #!/usr/bin/env bash
 
-# Usage example: ./download_release_artifacts.sh v1.8.0
+# Usage example: ./download_release_artifacts.sh 1.8.0
 
 set -Eeuo pipefail
 
 SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
 
-patches_path="${SCRIPT_DIR}/../patches"
-artifacts_path="${SCRIPT_DIR}/artifacts"
-mkdir -p "${patches_path}"
-mkdir -p "${artifacts_path}"
-# These files could in theory change from release to release, though their names should
-# be fairly stable.
-istio_files=(200-clusterrole 400-config-istio 500-controller 500-webhook-deployment 500-webhook-secret 500-webhook-service 600-mutating-webhook 600-validating-webhook)
-
-function download_ingress {
-  component=$1
-  version=$2
-  shift
-  shift
-
-  files=("$@")
-
-  component_dir="${artifacts_path}"
-  release_suffix="${version%?}0"
-  target_dir="${component_dir}"
-  rm -r "$component_dir"
-  mkdir -p "$target_dir"
-
-  for (( i=0; i<${#files[@]}; i++ ));
-  do
-    index=$(( i+1 ))
-    file="${files[$i]}.yaml"
-    target_file="$target_dir/$index-$file"
+# TODO: automatically detects the version via branch name or something.
+VERSION=$1
+
+istio_files=(200-clusterrole 400-config-istio 500-controller 500-webhook-deployment 500-webhook-secret 500-webhook-service 600-mutating-webhook 600-validating-webhook 0-networkpolicy-mesh)
```

Comment:
Are all of those files still needed? Most of those seem to be contained in `1-net-istio.yaml`.

Reply:
Yes, this list is used in the following line (around L22~).

```sh
  for file_prefix in "${istio_files[@]}"
  do
    for yaml in `find $dir -name "${file_prefix}.yaml" | sort`; do
      resolve_file "$yaml" "$resolved_file_name"
    done
```

The reason is that we need to exclude some files in `config` directory such as `202-gateway.yaml` and `203-local-gateway.yaml`.

Reply:
Hmm... I will re-write the script to `exclude` the `202-gateway.yaml` and `203-local-gateway.yaml` to avoid the confusion.